website/meta/universe.json

```diff
@@ -442,6 +514,80 @@
                 "website": "https://koaning.io"
             }
         },
+        {
+            "id": "Klayers",
+            "title": "Klayers",
+            "category": ["pipeline"],
+            "tags": ["AWS"],
+            "slogan": "Python Packages as AWS Lambda Layers",
+            "description": "A collection of Python Packages as AWS Lambda(λ) Layers",
+            "github": "keithrozario/Klayers",
+            "pip": "",
+            "url": "https://github.com/keithrozario/Klayers",
+            "code_language": "python",
+            "author": "Explosion",
+            "author_links": {
+                "twitter" : "https://twitter.com/explosion_ai",
+                "github": "https://github.com/explosion",
+                "website": "https://explosion.ai/"
+            },
```

Comment:
This doesn't seem right?

Reply:
Fix: updated the real author of the repository (copy-typo mistake)